kinova_driver/src/kinova_arm.cpp

```diff
@@ -300,6 +301,15 @@ KinovaArm::KinovaArm(KinovaComm& arm, const ros::NodeHandle& nodeHandle, const s
     ROS_ERROR("Missing param home position for actuator 6");
     ros::shutdown();
   }
+
+  KinovaAngles current_angles;
+  kinova_comm_.getJointAngles(current_angles);
+  ROS_INFO("Actuator 1 = [%f]", current_angles.Actuator1);
+  ROS_INFO("Actuator 2 = [%f]", current_angles.Actuator2);
+  ROS_INFO("Actuator 3 = [%f]", current_angles.Actuator3);
+  ROS_INFO("Actuator 4 = [%f]", current_angles.Actuator4);
+  ROS_INFO("Actuator 5 = [%f]", current_angles.Actuator5);
+  ROS_INFO("Actuator 6 = [%f]", current_angles.Actuator6);
```

Comment:
Do we need to print these here? The positions are published anyway, to my knowledge.

Reply:
Yeah, they are useful for developing, but I think they can be put as `ROS_DEBUG` or be excluded from the PR.